totp.js

```diff
@@ -0,0 +1,21 @@
+const crypto = require("crypto");
+
+const db = {
+  username: "oslevvce",
+  password: "123osl",
+  secrete: 1561115594546
+};
```

Comment:
See if there is a better way to store DB credentials than saving the sting in an opensource repo. Also is your DB local? I'd recommend not sharing the credentials out in the open, take care of spellings: oslevvce -> oslvvce. secrete -> secret.

Reply:
Sorry for the spelling mistake, yeah that credentials only for the testing purpose. And also I'm no doing anything with the that object.